alpha/lib/model/DeliveryProfile.php

```diff
@@ -315,12 +315,16 @@ protected function getFlavorAssetInfo($url, $urlPrefix = '', $flavor = null)
 		$ext = null;
 		$audioLanguage = null;
 		$audioLanguageName = null;
+		$label = null;
 		if ($flavor) {
 			if (is_callable(array($flavor, 'getFileExt'))) {
 				$ext = $flavor->getFileExt();
 			}
 			//Extract the audio language code from flavor
 			if ($flavor->hasTag(assetParams::TAG_AUDIO_ONLY)) {
+				if(is_callable(array($flavor, 'getLabel'))) {
+					$label = $flavor->getLabel();
```

Comment:
If getLabel is not callable that you will remain with a null value in the label param, shouldn't there be some kind of a default value? (like you have for "audioLanguage")